plugins/feeds/public/CertPaIT.py

```diff
@@ -1,28 +1,35 @@
+import re
 import logging
 from datetime import datetime, timedelta
+from dateutil import parser
 from core.observables import Hash
 from core.feed import Feed
 from core.errors import ObservableValidationError
 
 class CertPaIt(Feed):
 
     default_values = {
-        "frequency": timedelta(minutes=30),
-        "name": "CertPaIT",
-        "source" : "https://infosec.cert-pa.it/analyze/submission.rss",
-        "description": "This feed contains data from infosec.cert-pa.it",
+        'frequency': timedelta(minutes=30),
+        'name': 'CertPaIT',
+        'source' : 'https://infosec.cert-pa.it/analyze/submission.rss',
+        'description': 'This feed contains data from infosec.cert-pa.it',
     }
 
+    regex = '<p>Filename: <b>(?P<filename>.*)\\</b\\><br>Filetype: (?P<filetype>.*)</p>\n\t+<ul>\n\t+<li>IsDLL: (?P<isdll>[\\w]+)</li>\n\t\t\t\t\t\t\t\t\t<li>Packers: (?P<packers>[\\w]+)</li>\n\t\t\t\t\t\t\t\t\t<li>AntiDBG: (?P<antidbg>[\\w]+)</li>\n\t\t\t\t\t\t\t\t\t<li>AntiVM: (?P<antivm>[\\w]+)</li>\n\t\t\t\t\t\t\t\t\t<li>Signed: (?P<signed>[\\w]+)</li>\n\t\t\t\t\t\t\t\t\t<li>XOR: (?P<xored>[\\w]+)</li>\n\t\t\t\t\t\t\t\t</ul>'
```

Comment:
Maybe replace the `\t` and `\n` by `[^<]+` ?

The other alternative (cleaner and more reliable) is to have different regexes for each field, e.g.

```python
regex_packer = re.compile(r"Packers: (?P<packers>\w+)")
match = regex_packer.search(item['description'])
if match:
    # do things with match.group('packers')
```

I realize you're using groupdict which is pretty handy but this will become a pain to maintain if a single character or even field order changes in the feed.

Reply:
ok changing to many regexes